MessageViewController/MessageView.swift

```diff
@@ -273,13 +285,21 @@ public final class MessageView: UIView, MessageTextViewListener {
             + textViewHeight
             + (contentView?.bounds.height ?? 0)
     }
-
-    internal var textViewHeight: CGFloat {
-        return min(maxHeight, max(textView.font?.lineHeight ?? 0, textView.contentSize.height))
+    
+    internal var maxLineHeight: CGFloat {
+        return (font?.lineHeight ?? 0) * CGFloat(maxLineCount)
     }
-
+    
+    internal var maxScreenHeight: CGFloat {
+        return maxScreenRatio * (superview?.frame.height ?? 0) - heightOffset
```

Comment:
Isn't this using the kb height now w/ the offset? I agree that we shouldn't account for this at all. If anything, it should be optional.

Reply:
gotcha, will add this as another customization option